### PR TITLE
Added global exception handler and custom exceptions

### PR DESCRIPTION
**Pull request by ValentynSharshon**

### CODE REVIEW

File: src/main/java/app/exception/handler/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            return field + " " + message;
+        }
+        return e.getDefaultMessage();
+    }
+
+    private ResponseEntity<Object> getResponseEntity(HttpStatus status, Object error) {
+        ProblemDetail problemDetail = ProblemDetail.forStatus(status);
+        Map<String, Object> detail = new LinkedHashMap<>();
+        detail.put("error", error);
+        detail.put("timestamp", LocalDateTime.now().toString());
```
Comment (makaroshyna) [inline src/main/java/app/exception/handler/GlobalExceptionHandler.java:80]: I think it will be better to do constants of error and timestamp.

**Review by makaroshyna — APPROVED** [review]
Left one minor comment, but everything else looks good!

**Review by makaroshyna — APPROVED** [review]

File: src/main/java/app/exception/EntityNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package app.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String message) {
+        super(message);
+    }
+
+    public EntityNotFoundException(String message, Throwable cause) {
+        super(message, cause);
+    }
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/exception/EntityNotFoundException.java:10]: Just a prediction - we won't need it =)

File: src/main/java/app/exception/RegistrationException.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package app.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String message) {
+        super(message);
+    }
+
+    public RegistrationException(String message, Throwable cause) {
+        super(message, cause);
+    }
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/exception/RegistrationException.java:10]: same

**Review by VlasenkoYevhenii — APPROVED** [review]

**Pull request merged**